library/src/helpers/__tests__/schema.test.ts

```diff
@@ -27,4 +120,264 @@ describe('SchemaHelpers', () => {
+      const result = SchemaHelpers.humanizeConstraints(schema);
+      expect(result).toEqual(['multiple of 2']);
+    });
+
+    test('should handle number multipleOf', () => {
+      const schema = new Schema({ multipleOf: 1.5 });
+      const result = SchemaHelpers.humanizeConstraints(schema);
+      expect(result).toEqual(['multiple of 1.5']);
+    });
+
+    test('should handle decimal multipleOf', () => {
+      const schema = new Schema({ multipleOf: 0.0001 });
+      const result = SchemaHelpers.humanizeConstraints(schema);
+      expect(result).toEqual(['decimal places <= 4']);
+    });
+
+    test('should handle min length', () => {
+      const schema = new Schema({ minLength: 2 });
+      const result = SchemaHelpers.humanizeConstraints(schema);
+      expect(result).toEqual(['>= 2 characters']);
+    });
+
+    test('should handle max length', () => {
+      const schema = new Schema({ maxLength: 2 });
+      const result = SchemaHelpers.humanizeConstraints(schema);
+      expect(result).toEqual(['<= 2 characters']);
+    });
+
+    test('should handle min items', () => {
+      const schema = new Schema({ minItems: 2 });
+      const result = SchemaHelpers.humanizeConstraints(schema);
+      expect(result).toEqual(['>= 2 items']);
+    });
+
+    test('should handle max items', () => {
+      const schema = new Schema({ maxItems: 2 });
+      const result = SchemaHelpers.humanizeConstraints(schema);
+      expect(result).toEqual(['<= 2 items']);
+    });
+
+    test('should handle unique items', () => {
+      const schema = new Schema({ maxItems: 2, uniqueItems: true });
+      const result = SchemaHelpers.humanizeConstraints(schema);
+      expect(result).toEqual(['<= 2 unique items']);
+    });
+
+    test('should handle min properties', () => {
+      const schema = new Schema({ minProperties: 2 });
+      const result = SchemaHelpers.humanizeConstraints(schema);
+      expect(result).toEqual(['>= 2 properties']);
+    });
+
+    test('should handle max properties', () => {
+      const schema = new Schema({ maxProperties: 2 });
+      const result = SchemaHelpers.humanizeConstraints(schema);
+      expect(result).toEqual(['<= 2 properties']);
+    });
+
+    test('should handle not empty string', () => {
+      const schema = new Schema({ minLength: 1 });
+      const result = SchemaHelpers.humanizeConstraints(schema);
+      expect(result).toEqual(['non-empty']);
+    });
+
+    test('should handle not empty array', () => {
+      const schema = new Schema({ minItems: 1 });
+      const result = SchemaHelpers.humanizeConstraints(schema);
+      expect(result).toEqual(['non-empty']);
+    });
+
+    test('should handle not empty object', () => {
+      const schema = new Schema({ minProperties: 1 });
+      const result = SchemaHelpers.humanizeConstraints(schema);
+      expect(result).toEqual(['non-empty']);
+    });
+  });
+
+  describe('.isExpandable', () => {
+    test('object should be expandable', () => {
```

Comment:
What is not considered "expandable"? Since all these tests for `isExpandable` all results in true.

Reply:
Primitives without combined schemas :) I will add some tests. Thanks!